app/controllers/application_controller.rb

```diff
@@ -65,6 +65,11 @@ def confirm_two_factor_setup
       user.need_two_factor_authentication?(request) &&
       !user.totp_enabled?
 
-    redirect_to users_two_factor_authentication_setup_path
+    if Rails.configuration.enable_enhanced_2fa_experience
+      # TODO: Replace with route helper once fully migrated to enhanced 2FA.
+      redirect_to "/users/two_factor_authentication/setup"
```

Comment:
No, because there is an existing controller for which the route helper ends up being the same (i.e. `users/two_factor_authentication_setup`). Once we are completely off the existing 2FA flow, we can get rid of that controller, at which point the ruote helper will point to the new controller (i.e. `/users/two_factor_authentication/setup`).